memphis/templates/memphis_statefulset.yaml

```diff
@@ -580,7 +580,6 @@ spec:
             # During upgrades, healthz will be enabled instead to allow for a grace period
             # in case of JetStream enabled deployments to form quorum and streams to catch up.
             path: /healthz
```

Comment:
I was not sure if `/` or `/healthz` route is correct here. I adjust this on feedback :)

Reply:
Hi, thanks for paying attention to this. It supposes to be the `/` route.